packages/three-vrm-core/src/meta/VRMMetaLoaderPlugin.ts

```diff
@@ -48,7 +48,7 @@ export class VRMMetaLoaderPlugin implements GLTFLoaderPlugin {
   public constructor(parser: GLTFParser, options?: VRMMetaLoaderPluginOptions) {
     this.parser = parser;
 
-    this.needThumbnailImage = options?.needThumbnailImage ?? true;
+    this.needThumbnailImage = options?.needThumbnailImage ?? false;
```

Comment:
I've been wanted to fix this for a while. As a breaking and it's only doable during major update.